configs/benchmarks/classification/imagenet/vit-base-p16_ft-8xb256-coslr-100e_in1k.py

```diff
@@ -0,0 +1,121 @@
+# mmcls:: means we use the default settings from MMClassification
+_base_ = [
```

Comment:
Better inherit this [config](https://github.com/open-mmlab/mmselfsup/blob/dev-1.x/configs/benchmarks/classification/imagenet/vit-base-p16_ft-8xb128-coslr-100e_in1k.py), instead of rewriting all these modules.